.rubocop.yml

```diff
@@ -34,3 +35,6 @@ Style/FrozenStringLiteralComment:
 
 Lint/HandleExceptions:
   Enabled: false
+
+Rails/HttpPositionalArguments:
+  Enabled: false
```

Comment:
I don't have any opinion about this, so I'm OK with it.  If you have a minute, ask someone in DLSS-dev channel about what it means and whether we should disable it.